### PR TITLE
Update Readme and "Welcome" notebook

### PR DESCRIPTION
**Pull request by greazer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Please follow the [Python Extension ReadMe](https://github.com/microsoft/vscode-
 
 ## Working with other Languages
 
-The Jupyter Extension supports other languages such as Julia and R (as well as Python) in VS Code Insiders with our new Native Notebooks Experience!
+The Jupyter Extension supports other languages such as Julia and R in VS Code Insiders with our latest Native VS Code Notebooks Experience!
```
Comment (joyceerhl) [inline README.md:14]:
```suggestion
The Jupyter Extension supports languages such as Python, Julia and R in VS Code Insiders with our latest Native VS Code Notebooks Experience!
```

File: resources/startNativeNotebooks.ipynb [hunk 2]
```diff
@@ -6,7 +6,25 @@
    "source": [
     "# Welcome to the New VS Code Notebook Experience! \r\n",
     "\r\n",
-    "## To learn more about the new Notebook experience and get a tour around, click here: https://github.com/microsoft/vscode-jupyter/wiki/Introducing-Native-Notebooks"
+    "Whether you're new to the notebook experience in VS Code or a well-seasoned user, we'd like you to know that our experience has been revamped to provide:\r\n",
```
Comment (joyceerhl) [inline resources/startNativeNotebooks.ipynb:9]:
```suggestion
    "Whether you're new to the notebook experience in VS Code or a seasoned user, we'd like you to know that our experience has been revamped to provide:\r\n",
```

🧂?

Comment (greazer) [reply resources/startNativeNotebooks.ipynb]: Yeah, I don't even like seasoned... But I'm blanking on another term. But for now, I've removed 'well-' :)

**Review by DonJayamanne — APPROVED** [review]

**Review by joyceerhl — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Please follow the [Python Extension ReadMe](https://github.com/microsoft/vscode-
 
 ## Working with other Languages
 
-The Jupyter Extension supports other languages such as Julia and R (as well as Python) in VS Code Insiders with our new Native Notebooks Experience!
+The Jupyter Extension supports other languages in addition to Python such as Julia, R, and C# in VS Code Insiders with our latest Native VS Code Notebooks Experience!
```
Comment (IanMatthewHuff) [inline README.md:14]: Github won't let me place a comment there, but on line 8 there are two spaces between "partnership" and "of the" that could be fixed while we are fixing this.

File: README.md [hunk 4]
```diff
@@ -23,15 +23,11 @@ The Jupyter Extension supports other languages such as Julia and R (as well as P
 
 -   **Step 4.** Open or create a notebook file and start coding!
 
-- **Special Note:**  The Jupyter Extension in VS Code Insiders will include our Native Notebooks experience! To opt out:
+- **Special Note:**  The Jupyter Extension in VS Code Insiders will include our Native Notebooks experience by default! Because we are running in VS Code insiders and this build is updated every day, there may be times when our extension may fail to work at all. We do attempt to ensure that this doesn't happen frequently. If it does we strive to provide an updated extension build by the next business day. However, if you'd like to opt out of the native experience while working in VS Code insiders:
```
Comment (IanMatthewHuff) [inline README.md:26]: Capital I on "insiders" and a comma after "If it does"

File: resources/startNativeNotebooks.ipynb [hunk 5]
```diff
@@ -6,7 +6,25 @@
...
     "# Welcome to the New VS Code Notebook Experience! \r\n",
     "\r\n",
-    "## To learn more about the new Notebook experience and get a tour around, click here: https://github.com/microsoft/vscode-jupyter/wiki/Introducing-Native-Notebooks"
+    "Whether you're new to the notebook experience in VS Code or a seasoned user, we'd like you to know that our experience has been revamped to provide:\r\n",
+    "* Deep integration with the overall VS Code experience, including the ability to use editor extensions like VIM\r\n",
+    "* Extremely fast notebook load times\r\n",
+    "* A pleasing new look and feel\r\n",
+    "* Many bug fixes\r\n",
+    "\r\n",
+    "To learn more about this new Notebook experience and get a tour around, click here:  \r\n",
```
Comment (IanMatthewHuff) [inline resources/startNativeNotebooks.ipynb:15]: Lower case "notebook" here to match line 9

**Review by IanMatthewHuff — APPROVED** [review]
A few minor cleanup suggestions in here, but looks good.

File: README.md (hunk 3 above)
Comment (IanMatthewHuff) [reply README.md:14]: This is just an Ian suggestion but I'd rather be more sparing with "!". Doesn't seem super appropriate here.

File: README.md (hunk 4 above)
Comment (IanMatthewHuff) [reply README.md:26]: also this has what I would consider another extraneous exclamation point.

**Review by DavidKutu — APPROVED** [review]

**Pull request merged**